devtools/dumper-companion.py

```diff
@@ -392,7 +596,7 @@ def extract_partition(args: argparse.Namespace, vol) -> int:
             folders.append((upath, obj.mddate - 2082844800))
             continue
 
-        print(upath)
+        # print(upath)
```

Comment:
Could this be changed to an "if not silent: print(upath)"? Otherwise nothing is printed if not in silent mode